src/installation/guides/chroot.md

```diff
@@ -0,0 +1,222 @@
+> [mkfs.vfat(8)](https://man.voidlinux.org/mkfs.vfat.8).
+
+### Create a New Root and Mount Filesystems
+
+This guide will assume the new root filesystem is mounted on `/mnt`. You may
+wish to mount it elsewhere.
+
+If using UEFI, mount the EFI System Partition as `/mnt/boot/efi`.
+
+For example, if `/dev/sda2` is to be mounted as `/` and `/dev/sda3` is to be
+booted as `/home` and `dev/sda1` is the EFI System Partition
+
+```
+# mount /dev/sda2 /mnt/
+# mkdir /mnt/boot/
+# mount /dev/sda3 /mnt/home/
+# mkdir -p /mnt/boot/efi/
+# mount /dev/sda1 /mnt/boot/efi/
+```
+
+Initialize swap space, if desired using
+[mkswap(8)](https://man.voidlinux.org/mkswap.8).
+
+## Base Installation
+
+[Select a mirror to download packages
+from.](../../xbps/repositories/mirror/index.md) For simplicity, save this URL to
+a shell variable for later use. Append `/current` to this repository url to
+install a glibc distribution, append `/current/musl` for a musl distribution.
+
+> Note: musl binaries are not available for i686.
+
+For example, this is the url for glibc from the Tier 1 server in Finland,
+
+```
+# REPO=https://alpha.de.repo.voidlinux.org/current
+```
+
+xbps also needs to know what architecture is being installed, available options
+are `x86_64`, `x86_64-musl` and `i686` for PC architecture computers.
+
+For example,
+
+```
+# ARCH=x86_64
+```
+
+This architecture must be compatible with your current operating system, but
+does not need to be the same. If your host is running an x86_64 operating
+system, any of the three architectures can be installed (whether the host is
+musl or glibc), but an i686 host can only install i686 distributions.
+
+Use xbps to boot strap the distribution by installing the `base-system`
+metapackage.
+
+```
+# XBPS_ARCH=$ARCH xbps-install -S -r /mnt -R "$REPO" base-system
+```
+
+## Configuration
+
+### Entering the Chroot
+
+Mount the pseudo-filesystems needed for a chroot,
+
+```
+# mount -t proc /proc /mnt/proc
+# mount -t sysfs /sys /mnt/sys
+# mount -o bind /dev /mnt/dev
+# mount -t devpts pts /mnt/dev/pts
+```
+
+Copy the DNS configuration into the new root so that xbps can still download new
+packages inside the chroot,
+
+```
+# cp /etc/resolv.conf /mnt/etc/
+```
+
+Chroot into the new installation
+
+```
+# PS1='(chroot) # ' chroot /mnt/ /bin/bash
+```
+
+Write the hostname to `/mnt/etc/hostname`. Go through the options in
+[`/mnt/etc/rc.conf`](../../config/rc-files.md#rcconf). If installing a glibc
+distribution, edit `/mnt/etc/default/libc-locales`, uncommenting desired
+[locales](../../config/locales.md).
+
+[nvi(1)](https://man.voidlinux.org/nvi.1) is available in the chroot, but you
+may wish to install your preferred text editor at this time.
+
+[configure at least one super user account](../../config/users-and-groups.md).
+Other user accounts can be configured later, but there should either be a root
+password, or a new user account with [sudo(8)](https://man.voidlinux.org/sudo.8)
+privileges.
```

Comment:
We could explain a bit better what this means. This requires that the new account either be in the `root` group (bad - unsafe) or in the `wheel` group (good - safer). In the ROOTFS filesystem, it also required uncommenting this line in the `/etc/sudoers` file (seems like by installing `sudo` using the xbps method it's also required):

```
## Uncomment to allow members of group wheel to execute any command
# %wheel ALL=(ALL) ALL
```

PS: "configure" should be capitalized.

Reply:
This is something I think should be documented in the config doc that's linked. `passwd` is the easiest way to getting a system you can log into when you reboot, so that's the option that was shown explicitly.

Reply:
Very good point. I will try to update the documentation there.

Reply:
Here's what I've been working on, https://gist.github.com/jeffayle/cbc215551c57647e106031a7436dd76c

I figure it should go through its own pull request though

Reply:
Definitely its own PR! I think the groups table is an important piece of information too, so keeping it would be nice.

Reply:
yes, this would be added to the table that's already there!